src/sgame/sg_bot_ai.cpp

```diff
@@ -1109,10 +1109,10 @@ static AINodeStatus_t BotActionReachHealA( gentity_t *self )
 	int targetType = self->botMind->goal.getTargetedEntity()->s.modelindex;
 	vec3_t mins, maxs;
 	BG_BuildableBoundingBox( targetType, mins, maxs );
-	float dist = fabs( mins[0] );
+	float dist = RadiusFromBounds2D( mins, maxs );
```

Comment:
Creep and booster healing are based on center-to-center distance between the entities, as is GoalInRange. So bounding box sizes are not relevant and shouldn't be considered.